src/main/java/com/github/tomakehurst/wiremock/matching/ContainsPattern.java

```diff
@@ -26,6 +26,10 @@ public ContainsPattern(@JsonProperty("contains") String expectedValue, @JsonProp
         this.caseInsensitive = caseInsensitive;
     }
 
+    public ContainsPattern(@JsonProperty("contains") String expectedValue) {
```

Comment:
Fix incidental bug that was part of the original Wiremock: deserialization from JSON will fail without this constructor